envs/monkey_zoo/packer/setup_mimikatz_15.yml

```diff
@@ -0,0 +1,36 @@
+---
+- name: Create a new user
+  hosts: all
+  vars:
+    ansible_remote_tmp: C:\Windows\Temp
+  tasks:
+    - name: Create user
+      win_user:
+        name: m0nk3y
+        password: pAJfG56JX><
+        password_never_expires: yes
+        state: present
+        update_password: on_create
+        groups_action: add
+        groups:
+          - Administrators
+          - "Remote Desktop Users"
+
+    - name: Disable SMBv1
+      ansible.windows.win_optional_feature:
+        name: SMB1Protocol
+        state: absent
+
+    - name: Allow port 445 SMB
+      win_command:
+        cmd: netsh advfirewall firewall add rule name="Allow Port 445" dir=in action=allow protocol=TCP localport=445
```

Comment:
We need to disable powershell remoting ports as well?

Reply:
Talked about this via webex